set.mm

```diff
@@ -147154,6 +147148,398 @@ of an open range of nonnegative integers (of length equal to the length of
+      wb cword cv cpfx cop csubstr 3adant3 cun cfz elfzofz fzosplit syl raleqdv
+      eqwrd ralunb syl6bb eqidd biantrurd 3simpa elfzonn0 elfzo0le breq2 eqcoms
+      cn0 jca adantl mpbird pfxeq syl112anc bitr4d lencl anim12i 3adant2 ancomd
+      nn0red leidd 3ad2ant1 eqle sylan swrdspsleq bicomd anbi12d bitrd pm5.32da
+      cr ) DCUAZFZAWEFZBGDHIZJKZFZLZDAMZWHAHIZMZEUBZDIWOAIMZEWINZOZWNDBUCKABUCK
+      MZDBWHUDZUEKAWTUEKMZOZOWFWGWLWRTWJDECAUMUFWKWNWQXBWKWNOZWQWPEGBJKZNZWPEBW
+      HJKZNZOZXBXCWQWPEXDXFUGZNXHXCWPEWIXIWKWIXIMZWNWJWFXJWGWJBGWHUHKFXJBGWHUIG
+      WHBUJUKPQULWPEXDXFUNUOXCXEWSXGXAXCXEBBMZXEOZWSXCXKXEXCBUPUQXCWFWGOZBVCFZX
+      NOZBWHRSZBWMRSZWSXLTWKXMWNWFWGWJURQZWKXOWNWJWFXOWGWJXNXNBWHUSZXSVDPQWKXPW
+      NWJWFXPWGBWHUTPQZXCXQXPXTWNXQXPTZWKYAWMWHWMWHBRVAVBVEVFAEBBCDVGVHVIXCXMXN
+      WHVCFZOZWHWHRSZWHWMRSZXGXATXRWKYCWNWKYBXNWFWJYBXNOWGWFYBWJXNCDVJZXSVKVLVM
+      QWKYDWNWFWGYDWJWFWHWFWHYFVNZVOVPQWKWHWDFZWNYEWFWGYHWJYGVPWHWMVQVRXMYCYDYE
+      OLXAXGAEBWHCDVSVTVHWAWBWCWB $.
+  $}
+
+  $( Two (nonempty) words are equal if and only if they have the same prefix
+     and the same single symbol suffix.  (Contributed by Alexander van der
+     Vekens, 23-Sep-2018.)  (Revised by AV, 6-May-2020.) $)
+  pfxsuff1eqwrdeq $p |- ( ( W e. Word V /\ U e. Word V /\ 0 < ( # ` W ) )
+                          -> ( W = U <-> ( ( # ` W ) = ( # ` U )
+           /\ ( ( W prefix ( ( # ` W ) - 1 ) ) = ( U prefix ( ( # ` W ) - 1 ) )
+           /\ ( lastS ` W ) = ( lastS ` U ) ) ) ) ) $=
+    ( wcel cc0 chash cfv clt wbr w3a wceq c1 cmin co csubstr wa clsw wb syl cvv
+    cword cpfx cop cfzo cn wne hashgt0n0 lennncl 3adant2 fzo0end pfxsuffeqwrdeq
+    c0 syldan syld3an3 hashneq0 biimpd imdistani adantr swrdlsw 3anbi3d 3adant1
+    cs1 breq2 syl6bi impcom oveq1 id opeq12d oveq2d eqeq1d adantl eqeq12d fvexd
+    mpbird fvex s111 sylancl bitrd anbi2d pm5.32da ) CBUAZDZAWADZECFGZHIZJZCAKZ
+    WDAFGZKZCWDLMNZUBNAWJUBNKZCWJWDUCZONZAWLONZKZPZPZWIWKCQGZAQGZKZPZPWBWCWEWJE
+    WDUDNDZWGWQRWFWDUEDZXBWBWEXCWCWBWECULUFZXCCWAUGBCUHUMUIWDUJSAWJBCUKUNWFWIWP
+    XAWFWIPZWOWTWKXEWOWRVBZWSVBZKZWTXEWMXFWNXGXEWBXDPZWMXFKWFXIWIWBWEXIWCWBWEXD
+    WBWEXDCWAUOUPUQUIURBCUSSXEWNXGKZAWHLMNZWHUCZONZXGKZWIWFXNWIWFWBWCEWHHIZJZXN
+    WIWEXOWBWCWDWHEHVCUTXPWCAULUFZPZXNWCXOXRWBWCXOXQWCXOXQAWAUOUPUQVABAUSSVDVEW
+    IXJXNRWFWIWNXMXGWIWLXLAOWIWJXKWDWHWDWHLMVFWIVGVHVIVJVKVNVLXEWRTDWSTDXHWTRXE
+    CQVMAQVOTWRWSVPVQVRVSVTVR $.
+
+  ${
+    $d N y $.  $d V x $.  $d x y $.
+    $( Sets of words are disjoint if each set contains extensions of distinct
+       words of a fixed length.  (Contributed by AV, 29-Jul-2018.)  (Revised by
+       AV, 6-May-2020.) $)
+    disjwrdpfx $p |- Disj_ y e. W { x e. Word V | ( x prefix N ) = y } $=
+      ( cword cv cpfx co invdisjrab ) ABEDFAGCHIJ $.
+  $}
+
+  ${
+    $d x A $.  $d x S $.  $d x Y $.  $d x Z $.
+    $( Joining a prefix with an adjacent subword makes a longer prefix.
```

Comment:
Joining --> Concatenating
(I haven't checked other occurrences, but let's be consistent with "concatenate", "concatenation".)

Reply:
Done